pkg/controller/jobs/rayjob/rayjob_webhook.go

```diff
@@ -124,7 +124,7 @@ func (w *RayJobWebhook) validateCreate(job *rayv1.RayJob) (field.ErrorList, erro
 		clusterSpec := spec.RayClusterSpec
 		clusterSpecPath := specPath.Child("rayClusterSpec")
 
-		// Should not use auto scaler. Once the resources are reserved by queue the cluster should do it's best to use them.
+		// Should not use auto scaler. Once the resources are reserved by queue the cluster should do its best to use them.
 		if ptr.Deref(clusterSpec.EnableInTreeAutoscaling, false) {
 			allErrors = append(allErrors, field.Invalid(clusterSpecPath.Child("enableInTreeAutoscaling"), clusterSpec.EnableInTreeAutoscaling, "a kueue managed job should not use autoscaling"))
```

Comment:
Should this message inform users that they can switch on `EnableInTreeAutoscaling` if they turn this RayCluster into an Elastic job and they also enable `ElasticJobsViaWorkloadSlices` ?

Reply:
Hi @VassilisVassiliadis, Is there any reason why `enableInTreeAutoscaling` is still forbidden for RayJob?

Reply:
Hi @rueian I just didn't add support for RayJob objects in my PR. I only updated the code related to RayCluster objets. 

I can't think of a reason why the same approach wouldn't work for RayJob objects too.